frinx-uniconfig/user-guide/uniconfig-operations/uniconfig-properties/index.md

```diff
@@ -0,0 +1,15 @@
+# UniConfig Properties
+
+UniConfig properties are input parameters that can be used to configure the application.
```

Comment:
same comments as above:

- not input parameters
- there are actually 4 groups (DatabasePersistedProps , RuntimeMutableProps )

Reply:
rewritten